### ISSUE TITLE
Source

### ISSUE THREAD

**Issue by FailedCoder23** [sysplu/Fireflight#2]
May I modify the source code AS long as I give credit to you?

**Comment by TonicGaro** [comment]
Yeah just credit sysplu and TonicGaro, preferably have links to our GitHub.
